substratevm/src/com.oracle.svm.test/src/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "name" : "com.oracle.svm.test.jdk17.recordannotations.F",
```

Comment:
@jerboaa shouldn't this configuration file be moved under `substratevm/src/com.oracle.svm.test.jdk17`?

Reply:
@zakkak Maybe. Originally I had it there, but the build merges this then and combines `native-image` configs of the `com.orgacle.svm.test.jdk17` and `com.orgacle.svm.test` projects when it builds it into a native image. So hypothetically, if a `reflect-config.json` existed in `com.orgacle.svm.test` it would override it or it would at least be confusing which one it would indeed use. I can move it to `substratevm/src/com.oracle.svm.test.jdk17`.

Reply:
Good point. @olpaw can you please advise on this?